xds/internal/balancer/ringhash/ringhash.go

```diff
@@ -327,10 +327,11 @@ func (b *ringhashBalancer) ResolverError(err error) {
 // It triggers an update to cc when:
 // - the new state is TransientFailure, to update the error message
 //   - it's possible that this is a noop, but sending an extra update is easier
-//   than comparing errors
+//     than comparing errors
+//
 // - the aggregated state is changed
 //   - the same picker will be sent again, but this update may trigger a re-pick
-//   for some RPCs.
+//     for some RPCs.
```

Comment:
Interesting that gofmt didnt reformat this as single list. Should we just rewrite this comment as:

```golang
// UpdateSubConnState updates the per-SubConn state stored in the ring, and also
// the aggregated state.
//
// It triggers an update to cc when:
// - the new state is TransientFailure to update the error message. 
//   It's possible that this is a noop, but sending an extra update is easier
//   than comparing errors
// - the aggregated state has changed. The same picker will be sent again, 
//   but this update may trigger a re-pick for some RPCs.
```